packages/wonder-blocks-modal/util/get-nearest-modal-launcher-portal.test.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import {mount} from "enzyme";
+
+import getNearestModalLauncherPortal from "./get-nearest-modal-launcher-portal.js";
+import ModalLauncher from "../components/modal-launcher.js";
+import StandardModal from "../components/standard-modal.js";
+
+describe("getNearestModalLauncherPortal", () => {
+    test("when candidate is null, returns null", () => {
```

Comment:
Test case to ensure that we don't do anything bad if we're given a falsy element. Should probably rename to "when candidate is falsy, returns null".